docs/platforms/javascript/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -2,8 +2,6 @@
 title: Automatic Instrumentation
 description: "Learn what transactions are captured after tracing is enabled."
 sidebar_order: 10
-supported:
-  - javascript
```

Comment:
I think because of this we didn't actually show this page so far for all the browser guides 😅  e.g. https://docs.sentry.io/platforms/javascript/guides/react/performance/instrumentation/